build.gradle

```diff
@@ -41,7 +41,7 @@ repositories {
 
 dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.12'
-    implementation 'com.contrastsecurity:contrast-sdk-java:2.9'
+    implementation files('/Users/aliaandersen/Desktop/Projects/contrast-sdk-java/target/contrast-sdk-java-2.23-SNAPSHOT.jar')
     // https://mvnrepository.com/artifact/com.github.lgooddatepicker/LGoodDatePicker
```

Comment:
Ignore this for now. The sdk will have to be updated when a release is made. Current version: https://github.com/Contrast-Security-OSS/contrast-sdk-java/pull/66